src/views/list-all-favorites/list-all-favorites.js

```diff
@@ -10,12 +11,20 @@ export default {
     return {
       favorites: [],
       detailPath: DETAIL_PATH,
+      favoriteListFiltered: [],
     };
   },
   components: {
     ListPodcast,
   },
+  methods: {
+    async filterFavorites(term) {
+      const termSanitized = term.toLowerCase().trim();
```

Comment:
please move this logic inside 'filterBySearchTerm' function

Reply:
done